sdk/scheduler/src/test/java/com/mesosphere/sdk/framework/FrameworkRunnerTest.java

```diff
@@ -37,6 +37,8 @@
     public void beforeEach() {
         MockitoAnnotations.initMocks(this);
         Capabilities.overrideCapabilities(mockCapabilities);
+
+        when(mockSchedulerConfig.getMarathonName()).thenReturn("test-marathon");
```

Comment:
It's a little strange that this is needed here since none of the associated tests were changed. Does this indicate some kind of dependency that is not immediately apparent?